### ISSUE TITLE
[Accessibility] Checkboxes are unlabeled in `Export Profile...`

### ISSUE THREAD

**Issue by jooyoungseo** [microsoft/vscode#186857]
Type: <b>Bug</b>

1. Load NVDA/JAWS/Narrator.
1. In command palette, launch "Profiles: Export Profile..."
1. Tab to check boxes: they are unlabelled.

VS Code version: Code - Insiders 1.80.0-insider (39efe30bfc8e196f5be2119479092bd9eb83e174, 2023-06-30T14:32:07.986Z)
OS version: Windows_NT x64 10.0.22621
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|11th Gen Intel(R) Core(TM) i5-1145G7 @ 2.60GHz (8 x 2611)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|undefined|
|Memory (System)|15.71GB (6.49GB free)|
|Process Argv|--crash-reporter-id b05b88e5-8894-4031-ae34-fa034ebddea9|
|Screen Reader|yes|
|VM|0%|
</details><details><summary>Extensions (88)</summary>

Extension|Author (truncated)|Version
---|---|---
android-dev-ext|ade|1.3.2
Bookmarks|ale|13.3.1
openscad|Ant|1.1.1
spellright|ban|3.0.116
zoterolatex|bna|0.4.1
mermaid-markdown-syntax-highlighting|bpr|1.5.2
doxdocgen|csc|1.4.0
dscodegpt|Dan|2.1.13
vscode-markdownlint|Dav|0.51.0
vscode-eslint|dba|2.4.2
vscode-quick-select|dba|0.2.9
vscode-deno|den|3.19.1
gitlens|eam|14.0.1
EditorConfig|Edi|0.16.4
prettier-vscode|esb|9.19.0
vscode-google-translate|fun|1.4.13
codespaces|Git|1.14.8
copilot-chat|Git|0.3.2023062901
copilot-nightly|Git|1.94.222
remotehub|Git|0.58.0
vscode-github-actions|git|0.25.8
vscode-pull-request-github|Git|0.66.2
path-autocomplete|ion|1.24.1
latex-workshop|Jam|9.12.2
lilypond-syntax|jea|0.1.1
scheme|jea|0.2.0
better-cpp-syntax|jef|1.17.2
google-search|kam|0.0.1
vscode-lua-format|Koi|1.3.8
lilypond-formatter|lhl|0.2.3
lilypond-pdf-preview|lhl|0.2.8
lilypond-snippets|lhl|0.1.1
vslilypond|lhl|1.7.3
zotero|mbl|0.1.10
git-graph|mhu|1.30.0
vscode-docker|ms-|1.25.2
black-formatter|ms-|2023.2.0
flake8|ms-|2023.4.0
isort|ms-|2023.9.11781018
python|ms-|2023.10.1
vscode-pylance|ms-|2023.6.40
jupyter|ms-|2023.5.1101742258
jupyter-keymap|ms-|1.1.2
jupyter-renderers|ms-|1.0.17
vscode-jupyter-cell-tags|ms-|0.1.8
vscode-jupyter-slideshow|ms-|0.1.5
remote-containers|ms-|0.298.0
remote-ssh|ms-|0.102.0
remote-ssh-edit|ms-|0.86.0
remote-wsl|ms-|0.80.2
vscode-remote-extensionpack|ms-|0.24.0
azure-repos|ms-|0.34.0
cmake-tools|ms-|1.14.34
cpptools|ms-|1.16.3
cpptools-extension-pack|ms-|1.3.0
js-debug-nightly|ms-|2023.6.2717
remote-repositories|ms-|0.36.0
vscode-github-issue-notebooks|ms-|0.0.129
vscode-selfhost-test-provider|ms-|0.3.15
vscode-serial-monitor|ms-|0.10.0
vsliveshare|ms-|1.0.5873
resourcemonitor|mut|1.0.7
autodocstring|njp|0.6.1
pandocciter|not|0.10.2
shiny-python|Pos|0.1.2
quarto|qua|1.88.0
r-debugger|RDe|0.5.4
java|red|1.20.0
vscode-xml|red|0.25.0
r|REd|2.8.1
multi-command|ryu|1.6.0
vscode-deepl|soe|1.0.6
abc-music|sof|0.4.0
lua|sum|3.6.22
latex-utilities|tec|0.4.10
chatgpt|tim|1.1.2
cmake|twx|0.0.17
errorlens|use|3.11.1
intellicode-api-usage-examples|Vis|0.2.7
vscodeintellicode|Vis|1.2.30
vscode-arduino|vsc|0.6.0
vscode-java-debug|vsc|0.52.0
vscode-java-dependency|vsc|0.23.0
vscode-java-pack|vsc|0.25.12
vscode-java-test|vsc|0.39.0
vscode-maven|vsc|0.41.0
markdown-all-in-one|yzh|3.5.1
grammarly|znc|0.22.1

(1 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv695:30137379
vsins829:30139715
vsliv368:30146709
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
vstes627:30244334
vslsvsres303:30308271
pythontb:30258533
pythonptprofiler:30281269
vshan820:30294714
vscod805cf:30301675
bridge0708:30335490
bridge0723:30353136
vsaa593:30376534
pythonvs932:30404738
py29gd2263:30776702
vsclangdf:30492506
c4g48928:30535728
dsvsc012:30540252
pynewext54:30618038
pylantcb52:30590116
pyind779:30611226
pythonsymbol12:30651887
a9j8j154:30646983
showlangstatbar:30737417
pythonms35:30671666
ecj1e332:30687743
pythonfmttext:30716741
pythoncmvfstr:30726892
fixshowwlkth:30771523
hideindicator:30766887
pythongtdpath:30726887
bgfeh915:30780428
gsofa:30778558
pythonnosmt12:30779711
pythonidxpt:30768918
pythondjangots:30768917
pythonnoceb:30776497
copilotsettingt:30767686
e537b577:30772214
h0f32768:30779712
synctok:30776453
dsvsc013:30777762
dsvsc014:30777825

```

</details>

**Comment by meganrogge** [comment]
Interesting - they are announced on macOS with VoiceOver.

but I see they have no `aria-label` 
<img width="1527" alt="Screenshot 2023-07-03 at 8 15 36 AM" src="https://github.com/microsoft/vscode/assets/29464607/9e123538-d63d-4361-904b-5ca60521c1ef">
